RELEASE_NOTES.md

```diff
@@ -8,56 +8,31 @@ This document contains release notes for the current major release and all patch
 For prior releases,
 see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
-## 13.2.0 (2020-4-14)
+## 13.x.x (2020-5-xx)
```

Comment:
We need the 13.2.0 release notes in there. could you append to the existing document?

Reply:
Done.